styles.scss

```
@@ -269,24 +304,44 @@
 
   .nfd-ecommerce-standard-actions-container {
     display: grid;
-    grid: 155px / auto-flow 170px;
+    grid-template-columns: repeat(4, 170px);
+    grid-auto-rows: 155px;
     gap: 16px;
     padding-top: 32px;
+
+    @media (max-width: 1200px) {
+      grid-template-columns: repeat(3, 170px);
+    }
+    @media (max-width: 960px) {
+      grid-template-columns: repeat(2, 170px);
+    }
+    @media (max-width: 480px) {
+      grid-template-columns: repeat(1, 170px);
+    }
   }
 
   .nfd-ecommerce-minimal-tasks-container {
     display: grid;
-    grid: 78px / auto-flow 220px;
+    grid-template-columns: repeat(4, 220px);
```

We can use `auto-fit` and remove rules for every breakpoint.

```suggestion
    grid-template-columns: repeat(auto-fit, 220px);
```